build-oscript-swarm-agent.sh

```
@@ -20,13 +20,13 @@ if [ "${NO_CACHE}" = 'true' ] ; then
 fi
 
 docker build \
-	--pull \
+    --pull \
     $no_cache_arg \
```

Shellcheck (0.11.0)

[warning] no_cache_arg is referenced but not assigned.

(SC2154)